CMakeLists.txt

```diff
@@ -257,7 +270,7 @@ if(VELOX_ENABLE_PARQUET)
 endif()
 
 # define processor variable for conditional compilation
-if(${VELOX_CODEGEN_SUPPORT})
+if(${VELOX_ENABLE_CODEGEN_SUPPORT})
```

Comment:
I remember something about codegen not being used and should be removed? I'll check my issues to see if I can find it. But we can do that as a separate PR, just came to mind seeing the flag :)

Reply:
https://github.com/facebookincubator/velox/issues/4150#issuecomment-1698691986

Reply:
Yes, this is not really used, but we should have a discussion before actually removing it. I just renamed to follow the VELOX_ENABLE_.* pattern.